org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/DebugAdapter.java

```diff
@@ -187,26 +191,26 @@ public void sendEventLater(Events.DebugEvent event) {
 
     private void initialize() {
         // Register request handlers.
-        registerHandler(new InitializeRequestHandler());
-        registerHandler(new LaunchRequestHandler());
-        registerHandler(new AttachRequestHandler());
-        registerHandler(new ConfigurationDoneRequestHandler());
-        registerHandler(new DisconnectRequestHandler());
-        registerHandler(new SetBreakpointsRequestHandler());
-        registerHandler(new SetExceptionBreakpointsRequestHandler());
-        registerHandler(new SourceRequestHandler());
-        registerHandler(new ThreadsRequestHandler());
-        registerHandler(new StackTraceRequestHandler());
+        registerHandler(new InitializeRequestHandler(), null);
```

Comment:
don't modify this method signature.

Reply:
change a private method's signature does no harm?